dolphinscheduler-api/src/main/java/org/apache/dolphinscheduler/api/controller/ProjectController.java

```diff
@@ -265,7 +265,7 @@ public Result queryProjectCreatedAndAuthorizedByUser(@ApiIgnore @RequestAttribut
     @ApiException(LOGIN_USER_QUERY_PROJECT_LIST_PAGING_ERROR)
     @AccessLogAnnotation(ignoreRequestArgs = "loginUser")
     public Result queryAllProjectList(@ApiIgnore @RequestAttribute(value = Constants.SESSION_USER) User loginUser) {
-        Map<String, Object> result = projectService.queryAllProjectList();
+        Map<String, Object> result = projectService.queryAllProjectList(loginUser);
```

Comment:
there are some situations that we need the all projects without login user.
so i think this is no necessary.

Reply:
If this interface has no permission control, then you are right and this is no necessary, I have removed it in the new commit